scripts/idr0062_prediction.py

```diff
@@ -0,0 +1,206 @@
+#   are permitted provided that the following conditions are met:
+# 
+#   Redistributions of source code must retain the above copyright notice,
+#   this list of conditions and the following disclaimer.
+#   Redistributions in binary form must reproduce the above copyright notice, 
+#   this list of conditions and the following disclaimer in the documentation
+#   and/or other materials provided with the distribution.
+#   THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+#   ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED 
+#   WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED.
+#   IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT,
+#   INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY OR CONSEQUENTIAL DAMAGES (INCLUDING,
+#   BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA,
+#   OR PROFITS; OR BUSINESS INTERRUPTION)
+#   HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+#   OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS 
+#   SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+#
+# Version: 1.0
+#
+
+import numpy
+
+# omero
+from omero.gateway import BlitzGateway
+from omero_zarr import masks
+
+# stardist
+from stardist.models import StarDist2D
+from csbdeep.utils import normalize
+
+# geojson
+from geojson import Feature, FeatureCollection, Polygon
+import geojson
+
+
+# Connect to the server
+def connect(hostname, username, password):
+    """
+    Connect to the server
+    """
+    conn = BlitzGateway(username, password,
+                        host=hostname, secure=True)
+    print("Connected: %s" % conn.connect())
+    conn.c.enableKeepAlive(60)
+    return conn
+
+
+# Load-image
+def load_image(conn, image_id):
+    """
+    Load the Image object.
+    """
+    return conn.getObject('Image', image_id)
+
+
+# Load-Binary-from-server
+def load_binary_from_server(image):
+    """
+    Load the binary data as a TCZYX numpy array.
+    """
+    pixels = image.getPrimaryPixels()
+    size_z = image.getSizeZ()
+    size_c = image.getSizeC()
+    size_t = image.getSizeT()
+    size_y = image.getSizeY()
+    size_x = image.getSizeX()
+    z, t, c = 0, 0, 0  # first plane of the image
+
+    zct_list = []
+    for t in range(size_t):
+        for c in range(size_c):  # all channels
+            for z in range(size_z):  # get the Z-stack
+                zct_list.append((z, c, t))
+
+    values = []
+    # Load all the planes as YX numpy array
+    planes = pixels.getPlanes(zct_list)
+    s = "t:%s c:%s z:%s y:%s x:%s" % (size_t, size_c, size_z, size_y, size_x)
+    print(s)
+    print("Downloading image %s" % image.getName())
+    all_planes = numpy.stack(list(planes))
+    shape = (size_t, size_c, size_z, size_y, size_x)
+    return numpy.reshape(all_planes, newshape=shape)
```

Comment:
I guess the ``shape`` here means something like the shape of the image, not the 
``ROI > Shape`` 
If so, this is confusing if not read carefully, especially as this ``shape`` is used below, line 104 and lower, where the word

Reply:
Not the shape in the ``roi>shape`` sense but the shape/order in the number array. I have used a variable that numpy user will understand. I can change it to "order"